awc/src/frozen.rs

```diff
@@ -26,6 +26,8 @@ pub struct FrozenClientRequest {
     pub(crate) response_decompress: bool,
     pub(crate) timeout: Option<Duration>,
     pub(crate) config: ClientConfig,
+    // @TODO: find a way to avoid cloning
+    pub(crate) sni_host: Option<String>,
```

Comment:
I'm not sure here if we keep cloning this, or maybe have an Option<Rc<String>> ? (or maybe another borrowed / enum like it's done with ServerName for rustls ?)

Reply:
Fixed : i used a specific servername object which can be owned or borrowed